### PR TITLE
step-70: Rewrite the motivation for particles.

### PR DESCRIPTION
**Pull request by bangerth**
I had not actually quite understood that when I read through it first. I had to think about it a couple of days, and thought it's worthwhile making clearer.

### CODE REVIEW

**Comment by tamiko** [comment]
Travis complains with the following:
```
Error: repeated header title 'Representationofand' in file group 'examples/step-70/doc/intro.dox'
The command "./contrib/utilities/check_indentation.sh" exited with 123.
```

File: examples/step-70/doc/intro.dox [hunk 1]
```diff
@@ -426,27 +426,51 @@ of processor and the number of cells in $\Gamma$ increases, this leads
...
+<h4>Representation of Ω and Γ</h4>
+
+Remember that for both the penalization approach ($\mathcal{L}^2$ or $\mathcal{H}^1$)
+and the Nitsche method, we want to compute integrals that are approximated by
+the quadrature. That is, we need to compute
+\f[
+\beta (\textbf{v},\textbf{u})_{\Gamma} =
+\sum_{K\in \Gamma} \sum_{i=1}^{n_q}  \big(\hat{\textbf{u}}(\hat x_i)  (\textbf{v} \circ F_{K}) (\hat x_i) J_K (\hat x_i) w_i \big)
+\f]
+If you followed the discussion above, the you will recall that $\textbf{u}$
```
Comment (kronbichler) [inline examples/step-70/doc/intro.dox:439]:
```suggestion
If you followed the discussion above, you will recall that $\textbf{u}$
```

Comment (bangerth) [reply examples/step-70/doc/intro.dox:439]: I meant to say "then you will recall". Fixed.

**Comment by bangerth** [comment]
I think I fixed both issues.

**Pull request merged**